### ISSUE TITLE
Mekanism Infusion Support Broken

### ISSUE THREAD

**Issue by pupnewfster** [TagnumElite/ProjectE-Integration#17]
**Describe the bug**
I was finally getting around to testing switching my pack to using this when I noticed the following error in the console:
```
[11:27:01] [Server thread/ERROR] [projecteintegration]: Mapper 'Metallurgic Infuser' (com.tagnumelite.projecteintegration.plugins.PluginMekanism$MetallurgicInfuserMapper@71b830a9) Failed to run: java.lang.IllegalAccessError: tried to access field mekanism.common.InfuseStorage.type from class com.tagnumelite.projecteintegration.plugins.PluginMekanism$MetallurgicInfuserMapper
```
I believe this is due to in 9.7.2 we changed part of the internal API to more cleanly ensure that infusion information syncs better in the machines, when it runs out of a type of infusion. https://github.com/mekanism/Mekanism/commit/0dba7cb5841c9bc32b9c11a76f54f0ab97313142

Figured it was something that you might not be aware of and that other people may not notice from not looking at their console all that often.

**Comment by TagnumElite** [comment]
Thanks for this, might add a chat message if errors are experienced. Will look into this now.

**Comment by TagnumElite** [comment]
Should be fixed in 2.2.12.

**Comment by pupnewfster** [comment]
Works properly now thanks.
